Sources/TuistCore/Utils/FileHandler.swift

```diff
@@ -166,6 +199,14 @@ public final class FileHandler: FileHandling {
         }
     }
 
+    public func createSymbolicLink(_ path: AbsolutePath, destination: AbsolutePath) throws {
```

Comment:
Yeah, it's true - btw I have moved to hardlink because using soft link did not add any actual code to git history, just the file (so was useless for our purpose). Anyway, deleted all logic concerning symbolic link.